home_works/lesson14/file_worker.py

```diff
@@ -0,0 +1,25 @@
+from home_works.lesson14.handlers import JsonHandler, TxtHandler
+import os
+
+
+class FileWorker:
+    def __init__(self, path):
+        self.path = path
+        self.handler = None
+        if self.get_file_extension() == '.json':
```

Comment:
You can just set class dict like:
```Python
extensions_handler = {'.json': JsonHandler}
```
And here do some `self.handler = extensions_handler[self.get_file_extension()]`